### ISSUE TITLE
Starting an editor such as nano results in an error.

### ISSUE THREAD

**Issue by ghost** [romgrk/termrk#21]
Hello

Whenever I try to start nano, I get the following error:

```
TypeError: Cannot set property '0' of undefined
    at Terminal.write (/home/me/.atom/packages/Termrk/node_modules/term.js/src/term.js:1501:55)
    at /home/me/.atom/packages/Termrk/lib/termrk-view.coffee:214:33
    at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
    at Socket.<anonymous> (/home/me/.atom/packages/Termrk/lib/termrk-model.coffee:112:32)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:166:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:109:10)
    at Pipe.onread (net.js:517:20)
```

I first noticed this when I had outstanding commits in git and did a git pull, which resulted in other commits being merged, which invokes the active editor (export EDITOR=nano) to edit the merge commit message.

Thanks in advance

**Comment by romgrk** [comment]
Well noted. I will be looking into this.
(See note for delay https://github.com/romgrk/termrk/issues/18)

**Comment by ghost** [comment]
I think this can be closed, it seems to be solved with the latest 0.1.22 release.

**Comment by exside** [comment]
Works here as well on OS X 10.10
